vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopWallet.kt

```diff
@@ -355,7 +355,7 @@ class OidcSiopWallet(
             throw OAuth2Exception(Errors.INVALID_REQUEST)
         }
         return holder.createPresentation(
-            challenge = params.parameters.nonce,
+            challenge = params.parameters.nonce!!,
```

Comment:
please refactor to avoid !!
Possible solution: val nonce = params.parameters.nonce ?: run { Napier.w(...); throw ... }